app/assets/stylesheets/components/_alert.scss

```diff
@@ -0,0 +1,3 @@
+.alert-success { background-color: #e7f4e4 }
+.alert-danger { background-color: #f9dede }
+.alert-info { background-color: #e1f3f8 }
```

Comment:
The linter is also complaining about missing semicolons. Here is the full output from the Code Climate CLI:
```
1: Declaration should be terminated by a semicolon [scss-lint]
1: Color literals like `#e7f4e4` should only be used in variable declarations; they should be referred to via variable everywhere else. [scss-lint]
2: Declaration should be terminated by a semicolon [scss-lint]
2: Color literals like `#f9dede` should only be used in variable declarations; they should be referred to via variable everywhere else. [scss-lint]
3: Files should end with a trailing newline [scss-lint]
3: Declaration should be terminated by a semicolon [scss-lint]
3: Color literals like `#e1f3f8` should only be used in variable declarations; they should be referred to via variable everywhere else. [scss-lint]
```